### DISCUSSION TITLE
Testing zenon collective brainstorming twitter discussion proposal

### DISCUSSION THREAD

**Discussion by wizarddogc** [coinselor/twitter#10] [Tweet Ideas]
Zenon - The Network of Momentum 

We are intergalactic aliens sent to Earth for 2 missions.

1. Fuck Stacks in the pussy 

2. Fuck Lightning Network in the Ass 

Hope you join us on our mission!

Get Zenonized.

**Comment by sugoibtc** [comment]
based zenon alien

**Comment by coinselor** [comment]
Not PG13 enough.
